Documentation/development-flow.md

```diff
@@ -256,6 +256,24 @@ After the PR is open, you can make changes simply by pushing new commits. Your P
 
 ### Backport a Fix to a Release Branch
 
+#### Automated flow
```

Comment:
If the script will print the usage, could we just add a 1-liner to this doc to point to the script that will automate these steps? Then they can run the script to see its usage and we don't need completely different sections for automated and manual flow. I think it's important for the dev to first understand the process (manual flow), then second use tools to simplify his life. Alternatively, maybe we just switch the order of the automated and manual flow.

Reply:
Updated!